ui/raidboss/data/06-ew/raid/p5s.ts

```diff
@@ -152,6 +152,21 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    {
+      id: 'P5S Searing Ray',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '76[DF]7', source: 'Proto-Carbuncle', capture: false }),
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: Outputs.goFront,
```

Comment:
So, this is all I did for the normal mode. It could definitely be better, I don't know how well it translates either. However, I'm concerned about putting a "get behind" in the "don't get behind" callout. But, if we can figure out that wording I'm more than happy to swap it.

Reply:
Mmmm good point.